app/src/main/sentences/fr/media.yml

```diff
@@ -1,11 +1,14 @@
 play:
-  - (joue|jouer|jouez)|(mets|mais|mettez|mettre sur play) la|le chanson|média|piste
+  - (joue|jouer|jouez)|((re)?(mets|mais|mettez|mettre) sur play)| (la|le)? chanson|média|piste|musique|audio|film|stream
+  - démarre|démarrer|démarrez|allume|allumer|allumez le lecteur
+  - continue|continuer|continuez la? lecture
 
 pause:
-  - mais|mets|mettez|mettre (la|le chanson|média|piste sur pause)|(sur pause la|le chanson|média|piste)
+  - mais|mets|mettez|mettre ((la|le)? chanson|média|piste sur pause)|(sur pause (la|le)? chanson|média|piste|musique|audio|film|stream)
```

Comment:
You don't necessarily need the parenthesis in (la|le)?, but this is not important